packages/shared/src/components/modals/common/ModalHeader.tsx

```diff
@@ -37,8 +37,14 @@ export function ModalHeader({
   title,
   showCloseButton = true,
 }: ModalHeaderProps): ReactElement {
-  const { activeView, onRequestClose, tabs } = useContext(ModalPropsContext);
+  const { activeView, onRequestClose, tabs, isDrawer } =
+    useContext(ModalPropsContext);
   const modalTitle = title ?? (tabs ? activeView : undefined);
+
+  if (isDrawer) {
```

Comment:
Header is not needed on drawers (this component contains title and close button).